components/post_view/post_list_virtualized/post_list_virtualized.jsx

```diff
@@ -232,6 +293,16 @@ class PostList extends React.PureComponent {
         );
     }
 
+    handleShortcut = (e) => {
+        if (Utils.isKeyPressed(e, Constants.KeyCodes.ESCAPE)) {
+            if (this.state.showUnreadToast) {
```

Comment:
can both toasts be present at any given time? if so, shoudl we dismiss both or just one? is this the right order? I'm guessing that in case I can have both, the new messages one should be on top and I would expect to close that first, but I'd have to test that to be 100% sure

Reply:
Both toasts will not be present at the same time. Only one exists so i don't think the order matters here.